<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewGraphicStats</class>
 <widget class="QWidget" name="ViewGraphicStats">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Análise</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="observed">
        <attribute name="title">
         <string>Observado</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="PlotWindow" name="plotObsSessions" native="true"/>
         </item>
         <item row="1" column="0">
          <widget class="Line" name="line_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="PlotWindow" name="plotObsSubjects" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="expected">
        <attribute name="title">
         <string>Esperado</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="PlotWindow" name="plotExpSessions" native="true"/>
         </item>
         <item row="1" column="0">
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="PlotWindow" name="plotExpSubjects" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="residue">
        <attribute name="title">
         <string>Resíduo</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="PlotWindow" name="plotResSessions" native="true"/>
         </item>
         <item row="1" column="0">
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="PlotWindow" name="plotResSubjects" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pvalue">
        <attribute name="title">
         <string>P-Valor</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="PlotWindow" name="plotPvalSessions" native="true"/>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="PlotWindow" name="plotPvalSubjects" native="true"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QListWidget" name="list">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWindow</class>
   <extends>QWidget</extends>
   <header>plotwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
